aspnet/tutorials/first-web-api.rst

```diff
@@ -254,28 +301,44 @@ Update
 
 .. literalinclude:: first-web-api/sample/src/TodoApi/Controllers/TodoController.cs
   :language: c#
-  :lines: 44-60
+  :start-after: #region snippet_Update
+  :end-before: #endregion
   :dedent: 8
 
-``Update`` is similar to ``Create``, but uses HTTP PUT. The response is `204 (No Content) <http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html>`_.
+``Update`` is similar to ``Create``, but uses HTTP PUT. The response is `204 (No Content) <http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html>`__.
```

Comment:
You have to build in order to make the Update method work, but then you've lost the new items you created until then.